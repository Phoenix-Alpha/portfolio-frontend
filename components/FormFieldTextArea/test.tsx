import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FormFieldTextArea } from '.'

describe('<FormFieldTextArea />', () => {
  it('should renders correct <FormFieldTextArea />', () => {
    const wrapper = shallow(<FormFieldTextArea name="test" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
