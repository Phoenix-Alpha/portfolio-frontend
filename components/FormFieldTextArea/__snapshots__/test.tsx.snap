// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormFieldTextArea /> should renders correct <FormFieldTextArea /> 1`] = `
<Field
  name="test"
>
  <Component />
</Field>
`;
